cmd/substreams-sink-postgres/bulk_insert.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamingfast/cli"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/cli/sflags"
+	"github.com/streamingfast/shutter"
+	sink "github.com/streamingfast/substreams-sink"
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"github.com/streamingfast/substreams-sink-postgres/sinker"
+	"go.uber.org/zap"
+)
+
+var bulkLoadCmd = Command(bulkLoadE,
+	"bulk-load <psql_dsn> <endpoint> <manifest> <module> [<start>:<stop>]",
+	"Runs first load sink process",
+	ExactArgs(6),
+	Flags(func(flags *pflag.FlagSet) {
+		sink.AddFlagsToSet(flags)
+
+		flags.Int("flush-interval", 1000, "When in catch up mode, flush every N blocks")
+		flags.Uint64("bundle-size", 1000, "Size of output bundle, in blocks")
```

Comment:
`flush-interval` should be removed in favor of always using `bundle-size`, I don't see good reason to not align them to the same thing.

Reply:
Done